sycl/test/regression/usm_malloc_shared.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
I think it's overkill to run this test on all types of devices. Suggest removing GPU and ACC.

Reply:
Agreed as the changed don't propagate down to the diff device level.

Reply:
@jbrodman, Previously you asked me to add other device types such as CPU, GPU, HOST, ACC.
Since you changed your mind, I will remove them back.

Reply:
@romanovvlad Done.